### ISSUE TITLE
[BUG] ELSA 3.2 rc1 with latest studio. Null reference exception when editing JavaScript expressions

### ISSUE THREAD

**Issue by JochenSteinbrenner** [elsa-workflows/elsa-core#5606]
## Description
I have update my backend to 3.2 rc1 and studio sha-b496219 as docker container.
Before update (3.1.2) everthing runs fine.
After update, on server side everthing runs fine.
Designer comes up and shows workflows.
After editing any javasccript expression, the designer throws a null reference exception and can only be hard reloaded.

![ElsaException](https://github.com/elsa-workflows/elsa-core/assets/38906531/20fb6350-5bff-4b53-8f1c-d79262f8da51)

Chrome debug shows:
```
dotnet.runtime.8.0.6.0dujfiktpi.js:3 fail: Microsoft.AspNetCore.Components.Web.ErrorBoundary[0]
      System.NullReferenceException: Arg_NullReferenceException
         at Elsa.Api.Client.Extensions.JsonObjectExtensions.GetProperty(JsonObject model, String[] path)
         at Elsa.Api.Client.Extensions.JsonObjectExtensions.GetProperty[String](JsonObject model, String[] path)
         at Elsa.Api.Client.Extensions.ActivityExtensions.GetTypeName(JsonObject activity)
         at Elsa.Studio.Workflows.DiagramDesigners.Flowcharts.FlowchartDiagramDesignerProvider.GetSupportsActivity(JsonObject activity)
         at Elsa.Studio.Workflows.UI.Services.DefaultDiagramDesignerService.<>c__DisplayClass2_0.<GetDiagramDesigner>b__1(IDiagramDesignerProvider x)
         at System.Linq.Enumerable.TryGetFirst[IDiagramDesignerProvider](IEnumerable`1 source, Func`2 predicate, Boolean& found)
         at System.Linq.Enumerable.FirstOrDefault[IDiagramDesignerProvider](IEnumerable`1 source, Func`2 predicate)
         at Elsa.Studio.Workflows.UI.Services.DefaultDiagramDesignerService.GetDiagramDesigner(JsonObject activity)
         at Elsa.Studio.Workflows.Shared.Components.DiagramDesignerWrapper.OnInitializedAsync()
         at Microsoft.AspNetCore.Components.ComponentBase.RunInitAndSetParametersAsync()
         at Elsa.Studio.Components.StudioComponentBase.InvokeWithBlazorServiceContext(Func`1 func)
System.NullReferenceException: Arg_NullReferenceException
   at Elsa.Api.Client.Extensions.JsonObjectExtensions.GetProperty(JsonObject model, String[] path)
   at Elsa.Api.Client.Extensions.JsonObjectExtensions.GetProperty[String](JsonObject model, String[] path)
   at Elsa.Api.Client.Extensions.ActivityExtensions.GetTypeName(JsonObject activity)
   at Elsa.Studio.Workflows.DiagramDesigners.Flowcharts.FlowchartDiagramDesignerProvider.GetSupportsActivity(JsonObject activity)
   at Elsa.Studio.Workflows.UI.Services.DefaultDiagramDesignerService.<>c__DisplayClass2_0.<GetDiagramDesigner>b__1(IDiagramDesignerProvider x)
   at System.Linq.Enumerable.TryGetFirst[IDiagramDesignerProvider](IEnumerable`1 source, Func`2 predicate, Boolean& found)
   at System.Linq.Enumerable.FirstOrDefault[IDiagramDesignerProvider](IEnumerable`1 source, Func`2 predicate)
   at Elsa.Studio.Workflows.UI.Services.DefaultDiagramDesignerService.GetDiagramDesigner(JsonObject activity)
   at Elsa.Studio.Workflows.Shared.Components.DiagramDesignerWrapper.OnInitializedAsync()
   at Microsoft.AspNetCore.Components.ComponentBase.RunInitAndSetParametersAsync()
   at Elsa.Studio.Components.StudioComponentBase.InvokeWithBlazorServiceContext(Func`1 func)

```
The docker container of the studio shows no errors.
Backend shows no errors

Can you please give a little advice?

Regards Jochen Steinbrenner

## Steps to Reproduce
- Create new workflow
- Add run javascript activity
- Show and edit javascript block

**Comment by sfmskywalker** [comment]
Thanks for reporting! I get the same issue also when I run the Server + Studio image.

**Comment by sfmskywalker** [comment]
Fixed as of d70451e72b745b073ddd715cef4f6e888f409eb1
